apps/frontend/src/components/ETHBalance.tsx

```diff
@@ -20,7 +20,7 @@ export default function ETHBalance({ publicClient, account }: ETHBalanceProps) {
         });
         setBalance(balance);
       } catch (error) {
-        console.error('Error fetching balance:', error);
+
```

Comment:
No silent catch 🙏 if you really must remove the try and catch entirely. React can add a global component catcher in case